Lib/typing.py

```diff
@@ -3448,3 +3452,40 @@ def decorator(cls_or_fn):
         }
         return cls_or_fn
     return decorator
+
+
+
+def override(__arg: F) -> F:
```

Comment:
`typing_extensions` has to use this workaround since it supports Python 3.7, but `typing` can just use a positional-only arg. Might also be better to name it something more meaningful like `func` or `method`?